Build instructions fail under macOS "High Sierra"

The build instructions for h5cpp fail on macOS "High Sierra".
I have boost (1.67), CMake (3.11.4) and HDF5 (1.10.2) all installed using Homebrew.
My CMake command fails, with the following output:

<pre>
$  cmake ~/repos/h5cpp/
-- The CXX compiler identification is AppleClang 9.1.0.9020039
-- The C compiler identification is AppleClang 9.1.0.9020039
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Found conan: AUTO
-- Conan ** WARNING** : This detection of settings from cmake is experimental and incomplete. Please check 'conan.cmake' and contribute
-- Conan executing: conan install /Users/paterno/repos/h5cpp/conanfile_ess.txt -pr default --build=outdated
CMake Error at cmake/conan.cmake:322 (message):
  Conan install failed='No such file or directory'
Call Stack (most recent call first):
  cmake/conan.cmake:399 (conan_cmake_install)
  CMakeLists.txt:21 (conan_cmake_run)


-- Configuring incomplete, errors occurred!
See also "/Users/paterno/MyProjects/build-h5cpp/CMakeFiles/CMakeOutput.log".
</pre>

Since the failure appeared to be a missing dependency (conan), I also installed that (conan version 1.4.5) using Homebrew. After installing conan, and starting with a fresh build directory, I get a different error:
<pre>
$  cmake ~/repos/h5cpp/
-- The CXX compiler identification is AppleClang 9.1.0.9020039
-- The C compiler identification is AppleClang 9.1.0.9020039
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Found conan: AUTO
-- Conan ** WARNING** : This detection of settings from cmake is experimental and incomplete. Please check 'conan.cmake' and contribute
-- Conan executing: conan install /Users/paterno/repos/h5cpp/conanfile_ess.txt -pr default --build=outdated
Auto detecting your dev setup to initialize the default profile (/Users/paterno/.conan/profiles/default)
CC and CXX: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc, /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++
ERROR: Not able to automatically detect '/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc' version
ERROR: Unable to find a working compiler
Default settings
	os=Macos
	os_build=Macos
	arch=x86_64
	arch_build=x86_64
	build_type=Release
*** You can change them in /Users/paterno/.conan/profiles/default ***
*** Or override with -s compiler='other' -s ...s***


WARN: Remotes registry file missing, creating default one in /Users/paterno/.conan/registry.txt
cmake_findboost_modular/1.66.0@bincrafters/stable: Not found in local cache, looking in remotes...
cmake_findboost_modular/1.66.0@bincrafters/stable: Trying with 'conan-center'...
ERROR: Unable to find 'cmake_findboost_modular/1.66.0@bincrafters/stable' in remotes
CMake Error at cmake/conan.cmake:322 (message):
  Conan install failed='1'
Call Stack (most recent call first):
  cmake/conan.cmake:399 (conan_cmake_install)
  CMakeLists.txt:21 (conan_cmake_run)


-- Configuring incomplete, errors occurred!
</pre>

Am I doing something incorrectly?

--- a/README.md
+++ b/README.md
@@ -55,7 +55,7 @@ for the library, using it should be the same on each of these platforms.
 
 The minimum requirements for building the library are:
 
-* a C++ compiler, gcc>=4.8 should do well
+* a C++ compiler, gcc>=4.8.1 should do well
 * the boost libraries
 * the HDF5 C library (>=1.8.13 would do but >=1.10.0 is prefered)
 * cmake >= 3.0

--- a/README.md
+++ b/README.md
@@ -60,6 +60,19 @@ The minimum requirements for building the library are:
 * the HDF5 C library (>=1.8.13 would do but >=1.10.0 is prefered)
 * cmake >= 3.0
 
+The external library dependencies can be acquired and built using [Conan](https://conan.io/). Conan can be installed with PyPI: 
+```
+pip install conan
+```
+Then run the following to configure required Conan repositories:
+```
+conan remote add ess-dmsc https://api.bintray.com/conan/ess-dmsc/conan
+conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
+```
+and that's it, CMake will handle the rest!
+
+Alternatively you can manually install the dependencies to typical system locations. In this case please disable Conan by using the `-DCONAN=DISABLE` option when you run CMake. 
+
 Building the library is standard cmake & make fare, out of source. For example,
 in linux shell, you would do the following:
 

